common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -503,10 +503,18 @@ def teams_with_default_sort_order(self, teams):
 
     def verify_page_header(self):
         """Verify that the page header correctly reflects the current topic's name and description."""
-        self.assertEqual(self.browse_teams_page.header_topic_name, self.topic['name'])
-        self.assertEqual(self.browse_teams_page.header_topic_description, self.topic['description'])
+        self.assertEqual(self.browse_teams_page.header_name, self.topic['name'])
+        self.assertEqual(self.browse_teams_page.header_description, self.topic['description'])
 
-    def verify_on_page(self, page_num, total_teams, pagination_header_text, footer_visible):
+    def verify_search_header(self, search_results_page, search_query):
+        """Verify that the page header correctly reflects the current topic's name and description."""
+        self.assertEqual(search_results_page.header_name, 'Team Search')
+        self.assertEqual(
+            search_results_page.header_description,
+            'Showing results for "{search_query}"'.format(search_query=search_query)
+        )
+
+    def verify_on_page(self, teams_page, page_num, total_teams, pagination_header_text, footer_visible):
```

Comment:
Update the documentation for the `teams_page` argument. Also, is there any sort of type constraint for `teams_page`? It would be nice if there were some mixin which implemented the necessary methods. If each class requires different method implementations then I suppose there could be a common abstract base class, but maybe that's overengineering it somewhat...